src/Sylius/Bundle/AdminBundle/Resources/config/services/product/form.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sylius_admin.form.extension.product" class="Sylius\Bundle\AdminBundle\Form\Extension\ProductTypeExtension">
+            <tag name="form.type_extension" />
+        </service>
```

Comment:
Why did you extract only this service while a few others related to the product remained?